### PR TITLE
Test to verify RAS tools.

### PR DESCRIPTION
**Pull request by Pavithra1602**
Added few negetive cases for vpdupdate.

Signed-off-by: Pavithra <pavrampu@linux.vnet.ibm.com>

### CODE REVIEW

File: generic/ras_extended.py [hunk 1]
```diff
@@ -173,8 +173,23 @@ def test1_vpdupdate(self):
                                         "head -1", shell=True).strip()
         if path_db:
             copyfile_path = os.path.join(self.outputdir, 'vpd.db')
-            copyfile(path_db, copyfile_path)
+            shutil.copyfile(path_db, copyfile_path)
             self.run_cmd("vpdupdate --path=%s" % copyfile_path)
+        if os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/var/lib/lsvpd/run.vpdupdate'
+        elif os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
```
Comment (nasastry) [inline generic/ras_extended.py]: I don't see any difference between `if` condition file path and `elif` condition file path. Both are `/var/lib/lsvpd/run.vpdupdate`.

File: generic/ras_extended.py [hunk 2]
```diff
@@ -173,8 +173,23 @@ def test1_vpdupdate(self):
...
+            path = '/run/run.vpdupdate'
+        shutil.move(path, "/root/run.vpdupdate")
+        self.log.info("Running vpdupdate after removing run.vpdupdate")
+        self.run_cmd("vpdupdate")
+        shutil.move("/root/run.vpdupdate", path)
```
Comment (nasastry) [inline generic/ras_extended.py]: Many instances having files names hardcoded and the same used in different places. It's a good practice to use variable names.

File: generic/ras_extended.py [hunk 3]
```diff
@@ -173,8 +173,23 @@ def test1_vpdupdate(self):
...
+        process.run("rm -rf /var/lib/lsvpd/vpd.db; touch /var/lib/lsvpd/vpd.db",
```
Comment (nasastry) [inline generic/ras_extended.py]: `/var/lib/lsvpd/vpd.db` is a file and to remove file there is no need of `-r`.

**Comment by Pavithra1602** [comment]
Raised new PR.

**Pull request closed without merging**